cmd/ringbuf/src/lib.rs

```diff
@@ -154,12 +154,13 @@ fn ringbuf(
         if let Some(ref name) = subargs.name {
             if v.0.eq(name)
                 || (v.0.ends_with("RINGBUF")
+                    && (!v.0.ends_with("STRINGBUF"))
```

Comment:
woops. probably should use STRINGLOG or something ;)

Reply:
Perhaps, I don't think it is worth going back and changing everything though.  What do you think?

Reply:
yeah I think this is fine.